Components/lists.html

```diff
@@ -0,0 +1,186 @@
+                >Drafts
+              </li>
+              <li class="list-item flex-row-rev">
+                <ion-icon name="send-sharp" class="list-icon"></ion-icon>Sent
+              </li>
+              <li class="list-item flex-row-rev">
+                <ion-icon name="trash-sharp" class="list-icon"></ion-icon>Delete
+              </li>
+            </ul>
+          </div>
+
+          <h3 class="showCodeToggle" onclick="showCodeToggle()"></h3>
+
+          <div class="code hideCode">
+            <iframe
+              src="https://carbon.now.sh/embed/IiVLybkTpC3sGAi4yVyd"
+              style="
+                width: 100%;
+                height: 492px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
+          </div>
+        </div>
+
+        <!-- ** Ordered List  ** -->
+
+        <div class="exampleCont">
+          <h3 class="exampleText">Ordered List</h3>
+          <div class="flexCont flex-column">
+            <!--Big Number List-->
+
+            <ol class="xl-number-list">
+              <li class="xl-number-list-item">Shirt</li>
+              <li class="xl-number-list-item">Shoes</li>
+              <li class="xl-number-list-item">Pants</li>
+              <li class="xl-number-list-item">Watches</li>
+            </ol>
+          </div>
+          <div class="flexCont">
+            <!--Simple Number List-->
+
+            <ol class="number-list">
+              <li class="number-list-item">Shirt</li>
+              <li class="number-list-item">Shoes</li>
+              <li class="number-list-item">Pants</li>
+              <li class="number-list-item">Watches</li>
+            </ol>
+
+            <!-- Arrow List -->
+
+            <ol class="arrow-list">
+              <li class="arrow-list-item">Shirt</li>
+              <li class="arrow-list-item">Shoes</li>
+              <li class="arrow-list-item">Pants</li>
+              <li class="arrow-list-item">Watches</li>
+            </ol>
+
+            <!-- Diaamond List -->
+
+            <ol class="diamond-list">
+              <li class="diamond-list-item">Shirt</li>
+              <li class="diamond-list-item">Shoes</li>
+              <li class="diamond-list-item">Pants</li>
+              <li class="diamond-list-item">Watches</li>
+            </ol>
+          </div>
+
+          <h3 class="showCodeToggle" onclick="showCodeToggle()"></h3>
+
+          <div class="code hideCode">
+            <iframe
+              src="https://carbon.now.sh/embed/cDBrL7aPqFs2mPIIjmUo"
+              style="
+                width: 100%;
+                height: 739px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
```

Comment:
This iframe showing code snippits of Vertical nav insted of ordered list
![Screenshot (59)](https://user-images.githubusercontent.com/42497931/112653379-5c404b00-8e74-11eb-9346-f556540a33ab.png)

Reply:
Fixed